app/controllers/static_pages_controller.rb

```diff
@@ -1,5 +1,10 @@
 class StaticPagesController < ApplicationController
-  def home; end
+  def home
+    if logged_in?
```

Comment:
nên khởi tạo biến @feed_items ở action này
```suggestion
    @feed_items = []
    if logged_in?
```